runtime/compiler/env/j9method.cpp

```diff
@@ -1793,7 +1820,34 @@ TR_ResolvedRelocatableJ9Method::fieldAttributes(TR::Compilation * comp, int32_t
       if (comp->getOption(TR_DisableAOTInstanceFieldResolution))
```

Comment:
heh, really no reason other than "that's what the code looked like before". It's not a good answer, and, if you think it's something that should be fixed now rather than later I can make the necessary changes.